src/mol-plugin/behavior/dynamic/volume-streaming/util.ts

```diff
@@ -69,8 +69,15 @@ export async function getContourLevel(provider: 'emdb' | 'pdbe', plugin: PluginC
 export async function getContourLevelEmdb(plugin: PluginContext, taskCtx: RuntimeContext, emdbId: string) {
     const emdbHeaderServer = plugin.config.get(PluginConfig.VolumeStreaming.EmdbHeaderServer);
     const header = await plugin.fetch({ url: `${emdbHeaderServer}/${emdbId.toUpperCase()}/header/${emdbId.toLowerCase()}.xml`, type: 'xml' }).runInContext(taskCtx);
-    const map = header.getElementsByTagName('map')[0];
-    const contourLevel = parseFloat(map.getElementsByTagName('contourLevel')[0].textContent!);
+    const contours = header.getElementsByTagName('contour');
+
+    let primaryContour = contours[0];
+    for (let i = 1; i < contours.length; i++) {
+        if (contours[i].getAttribute('primary') === 'true') {
+            primaryContour = contours[i];
```

Comment:
I suppose you can `break` the loop here, right?

Reply:
Absolutely, added with [bbf4f1d](https://github.com/molstar/molstar/pull/377/commits/bbf4f1d1d30e13fe91d3905c3c7fd120cb8aa198).

Probably unrelated but FYI: maps released this week seem to be missing for PDBe (https://www.ebi.ac.uk/pdbe/entry/view3D/7n5d/?view=entry_index&viewer=litemol&assembly=1).